Article/Article.js

```diff
@@ -3,19 +3,29 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    console.log(`Dom element is of type ${this.domElement} and expandButton is of type ${this.expandButton}`);
     // Set a click handler on the expandButton reference, calling the expandArticle method.
+    this.expandButton.textContent = "expand";
     
+    this.expandButton.addEventListener("click", e => {
+      // expandArticle();
+      this.domElement.height = "400px";
```

Comment:
If you want the domElement to adjust for the height of each block of text, use auto. Also, you need to add a 'style' property to hook into that. It was an interesting idea though.

```suggestion
      this.domElement.style.height = "auto";
```